src/services/sync/strategies/TSP/TSPSchoolSyncer.js

```diff
@@ -542,8 +542,8 @@ class TSPSchoolSyncer extends mix(Syncer).with(ClassImporter) {
 				type: 'update-student',
 				entity: tspStudent.schuelerUid,
 				message:
-					`Schüler "${tspStudent.schuelerVorname} ${tspStudent.schuelerNachname}"` +
-					' konnte nicht aktualisiert werden.',
+					`student "${tspStudent.schuelerVorname} ${tspStudent.schuelerNachname}"` +
+					` with tsp id ${tspStudent.lehrerUid} and user id ${user._id} could not be updated.`,
```

Comment:
`tspStudent.lehrerUid` sounds wrong in case that is a student

Reply:
good catch, fixed